content/lang/header.en_template.php

```diff
@@ -64,5 +64,6 @@
 $LANG['NAV_IMG_LIB'] = 'Image Library';
 $LANG['NAV_IMG_CONTR'] = 'Image Contributors';
 $LANG['NAV_IMG_SEARCH'] = 'Image Search';
+$LANG['NAV_PUB_DAT_LIST'] = 'Public Datasets List'; 
```

Comment:
If this is not used by a lot of different pages, this should probably be in its own translation page.

Since such translation pages doesn't exist already for collections/datasets/publiclist.php, it seems a little silly to create them just for this one.

But. Leaving them in header.en_template.php seems like a bad pattern for us to have in the codebase.

I hate to say it, but let's create the translation files for just this one single translation...